README.md

```diff
@@ -72,7 +72,8 @@ please join us in
 
 ## Minimum Supported Rust Version (MSRV)
 
-This library should always compile with any combination of features on **Rust 1.29**.
+This library should always compile with any combination of features (minus
+`no-std`) on **Rust 1.29** or **Rust 1.47** with `no-std`.
```

Comment:
Can you change 1.29 to 1.41? :))

Reply:
Shouldn't bumping the regular MSRV be a separate PR?

I don't want to "sneak" an MSRV bump in this PR. I am willing to submit a new PR is this is desired.

Reply:
The MSRV change happened with #944 (ok, I'm being a bit philosophical here ... after we cut the 0.28 release, we decided that the MSRV for `master` would be 1.41). #961 makes this more explicit.

So it's not that this PR is sneaking in a MSRV bump; it's just reflecting one that happened while the PR was open.

Reply:
Fair enough.

Should I also update the CI workflow to check against 1.41 instead of 1.29?

Reply:
Good point you raise. I'll do it as a separate PR if you don't mind so we can get it merged post haste.

Reply:
#964

Reply:
Oh I've removed the pinning stuff in #946, but this PR uses it still and after re-reading @Kixunil comment I'm not sure if that is correct, we either need to:
1. Not remove the pinning stuff and leave this PR as is
2. Remove the pinning stuff in #964 and remove the pinning stuff from this one

Reply:
Oh and we are moving to `1.41.1` please mate not `1.41`, thanks.

Reply:
Hi @mcroad, just following up on this, sorry for all the confusion. This PR is correct as it is, you can ignore my comments above. If #946 merges first _then_ we can deal with removing the pinning stuff, otherwise if this PR merges first, I'll rebase #946 and handle the pinning stuff over there.

Reply:
I suggest merging #964 first, then rebasing this and resolving conflicts.